style/modules/header.css

```diff
@@ -0,0 +1,47 @@
+.header {
+    min-width: 500px;
+    display: block;
```

Comment:
this `display: block` isn't going to do anything because the `display: flex` immediately overrides it